src/service/nasa_meteor_service.js

```diff
@@ -0,0 +1,51 @@
+const axios = require('axios');
+const config = require('../config/env_config');
+const {getDate} = require('../util/date_util.js');
+const {mapMeteors} = require('../mapper/nasa_meteor_mapper');
+
+const getNasaMeteorsData = async (startDate, endDate, countOnly, wereDangerous) => {
+    const requestStartDate = startDate ? startDate : getDate().monday;
+    const requestEndDate = endDate ? endDate : getDate().friday;
+    return axios
+        .get(config.nasaApiConfig.url, {
+            params: {
+                start_date: requestStartDate,
+                end_date: requestEndDate,
+                api_key: config.nasaApiConfig.key,
+            },
+        })
+        .then((response) => {
+                if (countOnly) {
+                    return {count: countMeteors(response.data)};
+                }
+                if (wereDangerous) {
```

Comment:
What if both - countOnly & wereDangerous params are presented in request?

Reply:
Thank you for noticing. Fixed.